### DISCUSSION TITLE
I get an error that my MacBook can't identify the developer so I can't complete the installation

### DISCUSSION THREAD

**Discussion by Brandonhimself** [pawelsalawa/sqlitestudio#5426] [General]
I get an error that my MacBook can't identify the developer so I can't complete the installation

**Comment by tuffnatty** [comment]
See https://github.com/pawelsalawa/sqlitestudio/discussions/4451

**Comment by jacoblukewood** [comment]
The only solution that worked for me is ⌘ Command + ⌥ Option + O. This seems to do something different to just the standard right click and open that should work but doesn't.
